apps/api/lib/validations/booking.ts

```diff
@@ -6,7 +6,6 @@ import { extendedBookingCreateBody, iso8601 } from "@calcom/prisma/zod-utils";
 import { schemaQueryUserId } from "./shared/queryUserId";
 
 const schemaBookingBaseBodyParams = Booking.pick({
```

Comment:
Instead of updating this object which is the base object, let's instead update the `schemaBookingEditBodyParams` to omit the `uid` field when merging with this.